<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wTest</class>
 <widget class="QWidget" name="wTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>wTest</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_host">
       <property name="text">
        <string>Server IP,Port</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_time">
       <property name="text">
        <string>run times</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_stop">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>停止测试</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="browser1"/>
   </item>
   <item row="1" column="1">
    <widget class="QTextBrowser" name="browser2"/>
   </item>
   <item row="1" column="2">
    <widget class="QTextBrowser" name="browser3"/>
   </item>
   <item row="2" column="0">
    <widget class="QTextBrowser" name="browser4"/>
   </item>
   <item row="2" column="1">
    <widget class="QTextBrowser" name="browser5"/>
   </item>
   <item row="2" column="2">
    <widget class="QTextBrowser" name="browser6"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
